Okta/OktaAuth/OktaTokenManager.swift

```diff
@@ -93,6 +97,22 @@ open class OktaTokenManager: NSObject, NSCoding {
         
         return true
     }
+    
+    // Decodes the payload of a JWT
+    public static func decodeJWT(_ token: String) throws -> [String: Any]? {
+        let payload = token.split(separator: ".")
+        var encodedPayload = "\(payload[1])"
+        if encodedPayload.count % 4 != 0 {
+            let padding = 4 - encodedPayload.count % 4
+            encodedPayload += String(repeating: "=", count: padding)
+        }
+
+        guard let data = Data(base64Encoded: encodedPayload, options: []) else {
+            throw OktaError.JWTDecodeError
+        }
+        
+        return try? JSONSerialization.jsonObject(with: data, options: .mutableContainers) as! [String: Any]
```

Comment:
If the serialization fails, we're casting this to `nil` by using `try?`. Immediately after, we're going to force cast `nil` to a dictionary. We should perform validation here to ensure we do not crash the user's session.

Reply:
I understand that this is copy/paste from old implementation, but I would recommend to split this into 2 separate actions:
1. Try to parse json
2. Try to cast to Dict, as parser can return Any object - Dict, Array and etc.